spec.md

```diff
@@ -487,6 +512,96 @@ The tags MUST be in lexical order.
 
 When using the `last` query parameter, the `n` parameter is OPTIONAL.
 
+##### Listing Referrers
+
+*Note: this feature was added in distibution-spec 1.1.
```

Comment:
are we confirmed that this would only be in a point release?

Reply:
Yes!